{% extends "template.html" %}
{% set active_page = "cv" %}


{% block content %}

<!-- <div class='note__title'>  CV  </div> -->

<!-- <div class='note__content'> 

    <div class='note__title'>  Positions  </div>
 -->

<div class='note__title'> Degrees </div>
<div class="note"> 
    <div class='note__content__entry'>
    <div class='note__content__date'>  Feb 2019</div>
    Ph.D. degree with Summa cum laude in Bioinformatics, Center for Bioinformatics, Faculty of Science, University of Tübingen, Germany.
    </div>

    <div class='note__content__entry'>
    <div class='note__content__date'>  Nov 2013 </div>
    M.Sc. degree with Very Good in Macro-field of Study, Bioinfor- matics ans Systems Biology, Faculty of Mathematics, Informatics and Mechanics, University of Warsaw, Poland.
    </div>    

    <div class='note__content__entry'>
    <div class='note__content__date'> Sep 2011 </div>
    B.Sc. degree in Macro-field of Study, Bioinformatics and Systems
    Biology, Faculty of Mathematics, Informatics and Mechanics, University of Warsaw, Poland.
    </div>

</div>

<div class='note__title'>  Positions  </div>
<div class="note">
    

    <div class='note__content__entry'>
    <div class='note__content__date'> since April 2019 </div>
    Postdoctoral researcher in <a href="https://www.id-care.net/"> Infectious Diseases group</a> under a lead of Prof. Evelina Tacconelli, University of Verona, Italy.
    </div>

    <div class='note__content__entry'> 
    <div class='note__content__date'> Sep 2013 - Mar 2019 </div>
    Ph.D. student in <a href="https://uni-tuebingen.de/fakultaeten/mathematisch-naturwissenschaftliche-fakultaet/fachbereiche/informatik/lehrstuehle/algorithms-in-bioinformatics/"> Algorithms for Bioinformatics group</a> supervised by Prof. Daniel Huson, Center for Bioinformatics Tübingen. Ph.D. program  <a href="https://www.phd.tuebingen.mpg.de/imprs"> International Max Planck Research School From Molecules to Organisms </a>, Max Planck Institute for Developmental Biology and University of Tübingen, Germany.
    </div>

    <div class='note__content__entry'> 
    <div class='note__content__date'> 14 March - 20 April 2016 </div>
    Internship in <a href="https://pop-lab.org/"> Prof. Mihai Pop’s group </a>, Center for Bioinformatics and Computational Biology, University of Maryland, USA.
    </div>


    <div class='note__content__entry'> 
    <div class='note__content__date'> July – September 2012 </div>
    Internship in <a href="https://mccammon.ucsd.edu/"> Prof. Andrew J. McCammon’s group</a>, University of California San Diego, USA.
    </div>

    <div class='note__content__entry'> 
    <div class='note__content__date'> June 2011 - September 2013 </div>
    Graduate student fellowship in the TEAM project, co-founded by The Foundation for Polish Science at <a href='http://bionano.cent.uw.edu.pl/'> Biomolecular Machines Laboratory</a>, supervised by Prof. Joanna Trylska, Centre of New Technologies "Ochota", University of Warsaw, Poland.
    </div>

    <div class='note__content__entry'> 
    <div class='note__content__date'> 2009 - 2010 </div>
    Internship in <a href="http://genesilico.pl/"> Prof. Janusz Bujnicki’s group </a> at Laboratory of Bioinformatics and Protein Engineering, International Institute of Molecular and Cell Biology, Warsaw, RNA structure modeling.
    </div>
</div>

<div class='note__title'>  Teaching  </div>
    <div class="note">
            
        
        <div class='note__content__entry'>
        <div class='note__content__date'>  2021/2022 </div>
        A practical interdisciplinary PhD course on exploratory data analysis.
        </div>

        <div class='note__content__entry'>
        <div class='note__content__date'> Summer 2021 </div>
        A guest lecturer during the programming for bioinformatics about Metagenomics.
        </div>

        <div class='note__content__entry'>
        <div class='note__content__date'> 2017, 2018 </div>
        Informatics Vorkurs, Planned and carried out independently.
        </div>

        <div class='note__content__entry'>
        <div class='note__content__date'> 2017 </div>
        Bioinformatics pro-seminar, Carried out independently.
        </div>
        
        <div class='note__content__entry'>
        <div class='note__content__date'> 2017, 2018 </div>
        Informatics Vorkurs, Planned and carried out independently.
        </div>
        
        <div class='note__content__entry'>
        <div class='note__content__date'> 2016, 2017 </div>
        Visualization of Biological Data, TA-ed and planned.
        </div>

        <div class='note__content__entry'>
        <div class='note__content__date'> 2016 </div>
        Programming Projects in Python and Java, Carried out independently.
        </div>

        <div class='note__content__entry'>
        <div class='note__content__date'> 2014, 2016 </div>
        Bioinformatics I, TA-ed and planned.
        </div>

        <div class='note__content__entry'>
        <div class='note__content__date'> 2015 </div>
        Advanced Java for Bioinformatics, TA-ed and planned.
        </div>
    </div>

    <div class='note__title'>  Publications  </div>
    <div class="note">

    <div class='note__content__entry'>
    <div class='note__content__date'> 2022 </div>
    Gentilotti E.,  De Nardo P., Cremonini E., Górska A., Mazzaferri F., Canziani M. L., Hellou M. M., Olchowski Y.,  Poran I., Leeflang  M., Villacian J.,  Goossens  H., Paul M., Tacconelli E.,
    <a href="https://www.clinicalmicrobiologyandinfection.com/article/S1198-743X(21)00551-6/fulltext"> Diagnostic accuracy of point-of-care tests in acute community-acquired lower respiratory tract infections. A systematic review and meta-analysis</a>, Clinical Microbiology and Infection.
    </div>

    <div class='note__content__entry'>
    <div class='note__content__date'> 2020 </div>
     Hellou, M. M., Górska, A., Mazzaferri, F., Cremonini, E., Gentilotti, E., De Nardo, P., Paul, M.<a href="https://www.clinicalmicrobiologyandinfection.com/article/S1198-743X(20)30695-9/fulltext"> Nucleic-acid-amplification tests from respiratory samples for the diagnosis of coronavirus infections: systematic review and meta-analysis</a>, Clinical Microbiology and Infection.
    </div>

    <div class='note__content__entry'>
    <div class='note__content__date'> 2019 </div>
    Tacconelli, E., Górska A., Angelis, G., Lammens, C., Restuccia, G., Huson, D. H.,...Kazma, M. <a href="https://www.sciencedirect.com/science/article/pii/S1198743X19302472"> Estimating the Association between Antibiotic Exposure and Colonisation with Antibiotic-resistant Bacteria using Machine-learning Methods</a>, Journal of Antimicrobial Chemotherapy.
    </div>

    <div class='note__content__entry'>
    <div class='note__content__date'> 2019 </div>
    Arumugam K., Bağcı, C., Bessarab, I., Beier, S., Buchfink, B., Górska, A., Qiu, G., Huson, D. H., Williams R., <a href="https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-019-0665-y"> Annotated bacterial chromosomes from frame-shift-corrected long read metagenomic data</a>, Microbiome.
    </div>

    <div class='note__content__entry'>
    <div class='note__content__date'> 2018 </div>
    Górska, A., Peter, S., Willmann, M., Autenrieth, I., Schlaberg, R., Huson, D.H., 2018. <a href="https://www.sciencedirect.com/science/article/abs/pii/S1476927118301439?via%3Dihub"> Dynamics of the human gut phageome during antibiotics treatment</a>, Computational Biology and Chemistry.
    </div>

    <div class='note__content__entry'>
    <div class='note__content__date'> 2016 </div>
    Huson D., Beier S., Flade I., Górska A., El-Hadidi M., Mitra S., and others, <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004957"> MEGAN Community Edition - Interactive Exploration and Analysis of Large-Scale Microbiome Sequencing Data</a>, PLoS
    Computational Biology.
    </div>

    <div class='note__content__entry'>
    <div class='note__content__date'> 2016 </div>
    Górska A., Markowska-Zgrajek A., Równicki M., Trylska J. <a href="https://pubs.acs.org/doi/abs/10.1021/acs.jpcb.6b02081"> Scanning of 16S ribosomal RNA for peptide nucleic acid and targets</a>, The Journal of Physical Chemistry B.
    </div>

    <div class='note__content__entry'>
    <div class='note__content__date'> 2015 </div>
     Górska A., Jasiński M., and Trylska J., <a href="https://academic.oup.com/nar/article/43/17/e114/2414329"> MINT: Software to Identify Motifs and Short-Range Interactions in Trajectories of Nucleic Acids </a>, Nucleic Acids Research
    </div>

</div>
</div>

{% endblock %}

